### DISCUSSION TITLE
Charge resume settings

### DISCUSSION THREAD

**Discussion by pkeberly** [mr-manuel/venus-os_dbus-serialbattery#136] [Q&A]
From what I read in the config file and the parameters, it looks like SOC has to be less than 80% and a few other conditions to resume charging.  Does the driver use the calculated value as shown in the debug params or the BMS reported value?   I see CCL is 150A and my charger hopped right into bulk. Is my understanding wrong, or is my setup not following the rules...see below.  1218dev & v3.60~11

![20241220_204921](https://github.com/user-attachments/assets/524c567e-6a74-4efe-bd51-ec03cfb7172c)

![20241220_205001](https://github.com/user-attachments/assets/2d393561-3fab-4f80-8dcb-f36e392c7da5)

**Comment by mr-manuel** [comment]
You write your charger, so I assume you mean the lights on it? The lights do not match the driver state, since it hasn‘t this informations. Just ignore what state the charger is showing.
